resources/sass/pages/_committee-list.scss

```diff
@@ -0,0 +1,139 @@
+.ris-committee-list {
+    &-wrapper {
+        display: grid;
+        grid-template-columns: auto auto auto;
```

Comment:
this has to be tested in ie, because afaik the names for grid properties are different there

Reply:
did you test this on ie or is it not necessary for this case?